lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -361,17 +412,21 @@ export default function getRecipeInterface(
                     helpers,
                     inputRefreshToken,
                     antiCsrfToken,
-                    getRidFromHeader(req) !== undefined
+                    getRidFromHeader(req) !== undefined,
+                    disableAntiCSRF
```

Comment:
switch this boolean to instead use `inputTransferMethod` and `outputTransferMethod`